src/features/_links.scss

```diff
@@ -65,3 +65,11 @@ a {
     @extend %voidError;
   }
 }
+
+/* Anchors used as links have meaningful content */
+a[href]:not( [aria-label] ):not( [aria-labelledby] ):empty {
+  &::after {
+    @extend %contentError;
+    @include contentMessage(error, "Ensure that <a> has meaningful content or is labeled appropriately.");
```

Comment:
Typo here 😊 - "labelled"

Reply:
Ah damn! I'm always unsure which is correct :D Fixing now :) Thanks!

Reply:
That should do it. I spotted the same in the test.